src/logger/index.ts

```diff
@@ -486,36 +486,36 @@ class Logger {
           if (err) {
             // Handle error (e.g., file not found)
             console.error('Error reading file:', err)
-            res.status(500).send('Error reading file')
+            res.status(500).json({ error: 'Error reading file' })
```

Comment:
I think in the cases where we converted strings to json we could have posted:
res.type('txt')
in front of the call instead.
I am a bit concerned that there could be parts of the pipeline or tools that could break if we change the schema